my-project/src/Types/StoresContext.tsx

```diff
@@ -0,0 +1,81 @@
+import { createContext, useEffect, useState } from "react";
+
+type userStore = {
+  studentToken: string | null;
+};
+type facultyStore = {
+  facultyToken: string | null;
+};
+
+type store = {
+  adminToken: string | null;
+};
+
+export const UserContext = createContext<userStore>({
+  studentToken: null,
+});
+
+export const FacultyContext = createContext<facultyStore>({
+  facultyToken: null,
+});
+
+export const AdminContext = createContext<store>({
+  adminToken: null,
+});
+type UserProviderProps = {
+  children: React.ReactNode;
+};
+
+export const UserProvider: React.FC<UserProviderProps> = ({ children }) => {
+  const [studentToken, setStudentToken] = useState<string | null>(() => {
+    const storedToken = localStorage.getItem("studenttoken");
+    return storedToken || null;
+  });
+  useEffect(() => {
+    const storedToken = localStorage.getItem("studenttoken");
+    if (storedToken) {
+      setStudentToken(storedToken);
+    }
+  }, []);
+  return (
+    <UserContext.Provider value={{ studentToken }}>
+      {children}
+    </UserContext.Provider>
+  );
+};
+
+export const FacultyProvider: React.FC<UserProviderProps> = ({ children }) => {
+  const [facultyToken, setFacultyToken] = useState<string | null>(() => {
+    const storedToken = localStorage.getItem("facultytoken");
+    return storedToken || null;
+  });
+  useEffect(() => {
+    const storedToken = localStorage.getItem("facultytoken");
+    if (storedToken) {
+      setFacultyToken(storedToken);
+    }
+  }, []);
+  return (
+    <FacultyContext.Provider value={{ facultyToken }}>
+      {children}
+    </FacultyContext.Provider>
+  );
+};
```

Comment:
Faculty Store their name in local storage so clear the faculty name from local storage when faculty logouts

Reply:
### **Issue Resolved**
Resolved the local storage problem while logging out #5 
cleared Faculty name when faculty logs out

![Screenshot from 2023-08-06 22-40-32](https://github.com/20pa5a1210/Online-Exam-Final-Project-/assets/95855318/0c84c93c-1b1c-416f-ab07-3a72cda54cbe)